constraints/miden-vm/memory.air

```diff
@@ -0,0 +1,101 @@
+mod MemoryChipletAir
+
+### Helper functions/evaluators ###################################################################
+
+fn delta(v: scalar) -> scalar:
+    return v' - v
+
+
+# Enforces that columns must be zero.
+ev is_zero(main: [value[4]]):
+    enf v = 0 for v in value
+
+
+# Enforces that values in the columns in the current row must be equal to the values in the next 
+# row.
+ev equal_seq_lines(main: [value[4]]):
+    enf v' = v for v in value
+
+
+# Enforces that column must be binary.
```

Comment:
nit: Should we change this to `Enforces that the provided column must be binary.`?